bot/src/db.rs

```diff
@@ -86,6 +101,43 @@ pub struct EpochClassificationV1 {
 
     // Informational notes regarding this epoch
     pub notes: Vec<String>,
+
+    // Config values from Config struct
+    pub config: Option<EpochConfig>,
+
+    // General info about the Epoch
+    pub stats: Option<EpochStats>,
+}
+
+#[derive(Clone, Deserialize, Serialize)]
+pub struct EpochStats {
+    pub bonus_stake_amount: Option<u64>,
```

Comment:
Why the `Option` here?  Just wondering if 0 would be sufficient instead of None

Reply:
You're right; I made this a plain `u64` with a default value of `0`.